File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/strictmode/StrictModeActivity.java
package com.jdroid.android.sample.ui.strictmode;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class StrictModeActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return StrictModeFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/strictmode/StrictModeActivity.kt
package com.jdroid.android.sample.ui.strictmode

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class StrictModeActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return StrictModeFragment::class.java
    }
}
